src/meshapi/management/commands/replay_join_records.py

```diff
@@ -64,28 +92,69 @@ def handle(self, *args: Any, **options: Any) -> None:
 
         if not options["noinput"]:
             proceed_with_replay = input("Proceed with replay? (y/N): ")
-            if proceed_with_replay.lower() != "yes" and proceed_with_replay.lower() != "y":
+            if proceed_with_replay.lower() not in ["yes", "y"]:
                 print("Operation cancelled.")
                 return
 
+        if not join_records_to_replay:
+            logging.warning("Found no join records to replay. Quitting")
+            return
```

Comment:
nit: should this be before the confirmation prompt? Seems annoying to have to confirm for nothing to happen